Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -241,14 +241,14 @@ protected function getOpenInvoiceData($order): array
         // Discount cost
         if ($discountAmount != 0) {
 
-            $description = __('Total Discount');
-            $itemAmount = $this->adyenHelper->formatAmount($discountAmount, $currency);
+            $description = __('Discount');
+            $itemAmount = '-'.$this->adyenHelper->formatAmount($discountAmount, $currency);
```

Comment:
as this will be of type int shalle we do $number = -$this->adyenHelper->formatAmount($discountAmount, $currency); instead ?

Reply:
Yes it is working with the minus in front of the int